awslimits/server.py

```diff
@@ -51,6 +51,13 @@ def update_limit(limit_type):
     return redirect(url_for('limits'))
 
 
+@app.route("/healthcheck")
+def healthcheck():
+    """Healthcheck endpoint for confirming server is responding."""
+
+    return Response(status=200)
```

Comment:
```suggestion
    return Response('it works', status=200)
```
or any other message to make it a bit more friendly in case you want to open in the browser or use `curl`